mx.truffleruby/mx_truffleruby.py

```diff
@@ -127,7 +127,7 @@ def clean(self, forBuild=False):
             os.remove(self.launcher)
 
     def store_jvm_args(self):
-        distributions = [dep.name for dep in self.subject.buildDependencies]
+        distributions = [dep.name for dep in self.subject.buildDependencies] + ['tools:CHROMEINSPECTOR', 'tools:TRUFFLE_PROFILER']
```

Comment:
Is there any guarantee the tools are built when asking the classpath here, since they are no longer a dependency of any kind? Otherwise I think `mx` will complain and throw.